pyproject.toml

```diff
@@ -12,8 +12,7 @@ description = "A fork of the GYP build system for use in the Node.js projects"
 readme = "README.md"
 license = { file="LICENSE" }
 requires-python = ">=3.8"
-# The Python module "packaging" is vendored in the "pylib/packaging" directory to support Python >= 3.12.
-# dependencies = ["packaging>=23.1"]  # Uncomment this line if the vendored version is removed.
```

Comment:
Should we remove the vendored `packaging` source code as well?

Reply:
My preference would be to do this in two steps.